setup.py

```diff
@@ -19,9 +21,21 @@
       url='https://github.com/AtteqCom/zsl',
       license='MIT',
       install_requires=[
-        'flask_injector==0.8.0',
-        'future'
+          'future',
+          'Flask-Injector>=0.8' if _is_py3 else 'Flask-Injector==0.8.0',
+          'requests>=2.7',
+          'SQLAlchemy>=1',
```

Comment:
By the way, why do we use flask_injector? Maybe we could omit the FlaskInjector. Especially combined with the intention to omit the Flask dependency completely. We may use only the injector package. Also we do not use any special features of it. This may be the best how to deal with it, may it not?